server/services/forms/schemas/index.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     addTemplate: require('./add-template'),
     managePeople: require('./manage-people'),
     contributionRequest: require('./contribution-request'),
-    manageAppealDocuments: require('./manage-appeal-documents'),
+    manageAppealDocuments: require('./add-appeal-document'),
```

Comment:
Again... does the key here need to be updated?

Reply:
I have addressed this